src/eventDispatchers/mouseEventHandlers/keyDown.js

```diff
@@ -0,0 +1,34 @@
+import external from './../../externalModules.js';
+// State
+import { getters, state } from './../../store/index.js';
+// Todo: Where should these live?
+import getInteractiveToolsForElement from './../../store/getInteractiveToolsForElement.js';
+
+const cornerstone = external.cornerstone;
+
+/**
+ * KeyDown is used for keyboard input for mouse tools.
```

Comment:
You might be able to delete this and just create a new function by calling `.bind()` on `customCallbackHandler.js`

Reply:
This is a good idea. I can do this if we don't allow multiple functions to be bound to the same key, otherwise we need the bespoke routine. This is something to discuss.